integration_test/mysql/test_helper.exs

```diff
@@ -26,7 +26,8 @@ alias Ecto.Integration.TestRepo
 Application.put_env(:ecto, TestRepo,
   adapter: Ecto.Adapters.MySQL,
   url: "ecto://root@localhost/ecto_test",
-  pool: Ecto.Adapters.SQL.Sandbox)
+  pool: Ecto.Pools.Ownership.Server,
+  ownership_pool: pool)
```

Comment:
I would leave this as is, honestly. `inner_pool` sounds like an implementation detail.